src/Client.php

```diff
@@ -268,3 +285,12 @@ public function jsonPagingApiCall($method, $path, $params)
         return $result;
     }
 }
+
+
+class SleepService
```

Comment:
Just as more of an FYI than anything: it's generally a bad practice in a Composer-enabled library to put more than one class in a file. Composer uses directory and file paths in its autoloading. If someone wanted to test this `SleepService` they wouldn't be able to use `DuoAPI\SleepService` to get a new instance of it. In this case, since the functionality is only used in the `Client` class it's not as big of an issue.